Automattic/wp-calypso
# Performance Profiler: CWV Historical Graph Component

Related to https://github.com/Automattic/dotcom-forge/issues/8568

## Proposed Changes

* Add graph component for rendering history of CWV

## Testing Instructions



- Visit `/speed-test-tool?url=http%3A%2F%2Fexample.com`
- For each metric, check:
  - that the chart Y axis shows the correct number
  - hovering over the data points, the tooltip shows the correct numbers
  - the heading above the chart describes the correct tendency
  - the line color on the chart changes correctly when the value passes over a threshold
  - the last datapoint shows the same value as on the left scale

| Metric | Preview |
|--------|--------|
| FCP | ![CleanShot 2024-08-15 at 12 43 21@2x](https://github.com/user-attachments/assets/90363dd8-4437-427d-8f57-89fd24619d3d) |
| LCP | ![CleanShot 2024-08-15 at 12 43 41@2x](https://github.com/user-attachments/assets/c5f20d8b-1791-4d2f-8e9b-3342d2cd46e9) |
| CLS | ![CleanShot 2024-08-15 at 12 44 49@2x](https://github.com/user-attachments/assets/a4585872-8eb4-48ae-b158-6a6d00fba4fb) |
| INP | ![CleanShot 2024-08-15 at 12 44 57@2x](https://github.com/user-attachments/assets/e1cec6b9-666e-4cd9-a4de-cb149020a44e) |
| TTFB | ![CleanShot 2024-08-15 at 12 45 00@2x](https://github.com/user-attachments/assets/59795f77-93f8-4d39-a03c-cb6a75c56090) | 

## Pre-merge Checklist
